woocommerce/woocommerce
# fix: CYS - update copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR updates the copy according to the Figma mockup.

| Before | After |
|--------|--------|
| ![image](https://github.com/woocommerce/woocommerce/assets/4463174/3b95ffed-2f57-4c0e-9fc6-d298730cc8b1) | ![image](https://github.com/woocommerce/woocommerce/assets/4463174/9734cfab-6930-47cf-813b-ac1db71767e3) |
|![image](https://github.com/woocommerce/woocommerce/assets/4463174/30ab98fd-d39c-45c7-bda3-1a0c346b6897)| ![image](https://github.com/woocommerce/woocommerce/assets/4463174/4cd90245-cff5-48d6-ab20-6c26eecb66c0) |
|![image](https://github.com/woocommerce/woocommerce/assets/4463174/1a8bd530-e927-46a5-bbfb-8ce60e980cc2)|![image](https://github.com/woocommerce/woocommerce/assets/4463174/043ad946-bfd7-4daf-a0d4-20379c68da2b)|
|![image](https://github.com/woocommerce/woocommerce/assets/4463174/5a85a5fb-e1e8-4fed-87ae-cb517ff6c2e2)| ![image](https://github.com/woocommerce/woocommerce/assets/4463174/b3e20c35-9d7e-4284-8532-33f3f00057ec) |
| ![image](https://github.com/woocommerce/woocommerce/assets/4463174/7d03d7a3-37b3-45d0-a088-26ab64e33716) | ![image](https://github.com/woocommerce/woocommerce/assets/4463174/1f6da941-b0a9-430e-a111-faf056cd072b) | 


> [!WARNING]  
> I noticed that the mockup doesn't have the: `Powered by experimental AI. Learn more.`. Should we remove it?

| Current implementation | Mockup |
|--------|--------|
| ![image](https://github.com/woocommerce/woocommerce/assets/4463174/299e13b2-2331-4f10-a58b-983ca83f88c7) | ![image](https://github.com/woocommerce/woocommerce/assets/4463174/c5665004-4f92-48b2-bb53-0443f3bd46cb) | 






Closes #42698.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new site either via local environment or JN. Make sure to install WooCommerce Beta Tester as well.
2. Enable `customize-store` feature flag in `Tools -> WCA Test Helper -> Features`
3. Go to `WooCommerce -> Home` and click on `Customize your store` task.
4. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change `status.indicator` to `major`. 
5. Refresh the page, go through the process.
6. Ensure that the copy matches with the screenshots.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS: Update copy.

#### Comment 

</details>
